about.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/162dfb3961.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <title>Capstone Project1</title>
+  </head>
+  <body>
+    <header class="desktop-nav-header">
+      <div class="top-nav">
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-facebook-f"></i>
+        </a>
+        <a href="#" class="top-nav-link">
+          <i class="fa-brands fa-twitter"></i>
+        </a>
+        <a href="#" class="top-nav-link"> English </a>
+        <a href="#" class="top-nav-link"> myPage </a>
+        <a href="#" class="top-nav-link last"> logout </a>
+      </div>
+      <nav class="desktop">
+        <div class="desktop-logo">
+          <a href="index.html" class="desk-logo">
+            <img src="images/logo.png" alt="logo" />
+          </a>
+        </div>
+        <div class="desktop-nav">
+          <ul class="desktop-nav-lists">
+            <li class="nav-list">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="#program">Program</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html">Speakers</a>
+            </li>
+            <li class="nav-list">
+              <a class="nav-link" href="index.html#Sponsors">Sponsor</a>
+            </li>
+            <li class="nav-list">
+              <a href="">
+                <button class="nav-btn btn">Campaign</button>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <!-- Hero section -->
+    <section class="hero-about" id="home">
+      <div class="nav-toggle">
+        <div class="hamburger bar1"></div>
+        <div class="hamburger bar2"></div>
+        <div class="hamburger bar3"></div>
+      </div>
+      <nav class="mobile-nav">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="index.html" class="nav-link">Home</a>
+          </li>
+          <li class="nav-item">
+            <a href="about.html" class="nav-link">about</a>
+          </li>
+          <li class="nav-item">
+            <a href="#program" class="nav-link">program</a>
+          </li>
+          <li class="nav-item">
+            <a href="#speakers" class="nav-link">speakers</a>
+          </li>
```

Comment:
- [ ] **[OPTIONAL]** kindly capitalize the words `About, Program, Speakers` for a better UX. 👍🏽 

![image](https://user-images.githubusercontent.com/58642949/190403382-3f9a227c-77c2-479f-8d0d-fda767d21ff5.png)